src/git/config.go

```diff
@@ -252,15 +252,15 @@ func UpdateShouldHackPush(value bool) {
 
 // Helpers
 
-func getConfigurationValue(key string) (result string) {
+func getLocalConfigurationValue(key string) (result string) {
```

Comment:
For this rename, I think we also want to update the process of getting the result to only look at the local config.